docs/3.0.0-beta.x/advanced/customize-admin.md

```diff
@@ -33,9 +33,101 @@ By default, the administration panel is exposed via [http://localhost:1337/admin
 
 The panel will be available through [http://localhost:1337/dashboard](http://localhost:1337/dashboard) with the configurations above.
 
+---
+
 ### Development mode
 
-**_Currently not available_**
+To enable the front-end development mode you need to start your application using the `--watch-admin` flag.
+
+```bash
+cd my-app
+strapi develop --watch-admin
+```
+
+With this option you can do the following:
+
+#### Customize the `strapi-admin` package
+
+All files added in `my-app/admin/src/` will either be replaced or added
+
+**Example: Changing the available locales of your application**
+
+```bash
+# Create both the admin and admin/src folders
+cd my-app && mkdir -p admin/src
+# Change the available locales of the administration panel
+touch admin/src/i18n.js
+```
+
+**Path: `my-app/admin/src/i18n.js**
+
+```js
+import { addLocaleData } from 'react-intl';
+import { reduce } from 'lodash';
+import en from 'react-intl/locale-data/en';
+import fr from 'react-intl/locale-data/fr';
+
+// We dismiss pt-BR and zh-Hans locales since they are not supported by react-intl
+const locales = {
+  en,
+  fr,
+};
+const languages = Object.keys(trads);
+
+/**
+ * Dynamically generate `translationsMessages object`.
+ */
+const translationMessages = reduce(
+  languages,
+  (result, language) => {
+    const obj = result;
+    obj[language] = trads[language];
+
+    if (locales[language]) {
+      addLocaleData(locales[language]);
+    }
+
+    return obj;
+  },
+  {}
+);
+
+export { languages, translationMessages };
+```
+
+> With this modification only English and French will be available in your admin
+
+#### Customize a plugin
+
+Similarly to the back-end override system any file added in `my-app/extensions/<plugin-name>/admin/` will be copied and used instead of the original one (use with care).
+
+**Example: Changing the current WYSIWYG**
+
+```bash
+cd my-app/extensions
+# Create the content manager folder
+mkdir content-manager && cd content-manager
+# Create the admin folder
+mkdir -p admin/src
+# Create the components folder and the WysiwygWithErrors one
+cd admin/src && mkdir -p components/WysiwygWithErrors
+# Create the index.js so the original file is overridden
+touch components/WysiwygWithErrors/index.js
+```
+
+**Path: `my-app/extensions/content-manager/admin/src/components/WysiwygWithErrors/index.js**
```

Comment:
Here is a missing **`** at the end of the line.

Reply:
And can you follow this format please https://github.com/strapi/strapi/pull/4152/files#diff-67f3303196b46a0cd7c1d31b87dfca5dR19